platform/ios/scripts/package.sh

```diff
@@ -117,7 +117,7 @@ if [[ ${BUILD_FOR_DEVICE} == true ]]; then
             -o ${OUTPUT}/static/${NAME}.framework/${NAME} \
             ${LIBS[@]/#/${PRODUCTS}/${BUILDTYPE}-iphoneos/lib} \
             ${LIBS[@]/#/${PRODUCTS}/${BUILDTYPE}-iphonesimulator/lib} \
-            `find mason_packages/ios-${IOS_SDK_VERSION} -type f -name libgeojson.a`
+            `cmake -LA -N ${DERIVED_DATA} | grep MASON_PACKAGE_icu_LIBRARIES | cut -d= -f2`
```

Comment:
I like looking up the value in the cmake cache because it automatically gets the correct path that (I think) is a result of `mason_use(icu VERSION 58.1)` and `target_add_mason_package(mbgl-core PUBLIC icu)` in the `config.cmake` file in the iOS platform directory. This way, this package.sh script does not need to know anything about the platform (although that would not be bad), the mason platform version, or the version of ICU. That can all come from the result of cmake and its config.cmake.